cuda_core/cuda/core/experimental/_device.py

```diff
@@ -74,19 +115,21 @@ def uuid(self) -> str:
 
     @property
     def name(self) -> str:
-        # assuming a GPU name is less than 128 characters...
-        name = handle_return(cuda.cuDeviceGetName(128, self._id))
+        """Return the device name."""
+        # CUDA Runtime uses up to 256 characters, use the same for consistency
```

Comment:
mega-nit : "... characters. Use the same for consistency."

Reply:
Reworded it so that it's a single sentence. Done.